flixel/system/debug/interaction/tools/Pointer.hx

```diff
@@ -21,13 +21,17 @@ class GraphicCursorCross extends BitmapData {}
  */
 class Pointer extends Tool
 {
+	public var _selectionArea:FlxRect = new FlxRect();
+
 	var _selectionStartPoint:FlxPoint = new FlxPoint();
 	var _selectionEndPoint:FlxPoint = new FlxPoint();
 	var _selectionHappening:Bool = false;
 	var _selectionCancelled:Bool = false;
-	var _selectionArea:FlxRect = new FlxRect();
+	var _selectedItem:FlxBasic;
```

Comment:
this field is set but never used

Reply:
Same with this, forgot to remove old debugging stuff